Data Review Modal

melcidar opened the pull request:
--------------------------------------------------------------------
 MAIN
 * added modal for data review 
 OTHER
 * have to edit Team Members to show names of developers, status and duration
 * 
<img width="947" alt="Screenshot 2023-06-29 111249" src="https://github.com/becarselman/ant-colony-pharaoh-frontend/assets/119449806/62597658-3d5a-4c3c-8809-b7be5005790c">

melcidar committed:
Data Review Modal

--------------------------------------------------------------------
 MAIN
 * added modal for data review
--------------------------------------------------------------------
 OTHER
 * have to edit Team Members to show names of developers, and status
--------------------------------------------------------------------

becarselman commented on src/components/DataModal/DataInfo.js:
```diff
@@ -0,0 +1,49 @@
+          {InfoItems}
+        </div>
+        <div className='button-container'>
+          <div className='delete-button'>Delete Project</div>
```
If this component need to be reusable Delete Project will not suite all cases

becarselman replied:
And if it's not need to be reusable then you need to rename component in something similar to ProjectOverview

becarselman commented on src/components/DataModal/DataInfo.scss:
```diff
@@ -0,0 +1,147 @@
+};
+
+.button-container{
+    //padding: 24px;
```
remove commented code after you finish

becarselman commented on src/components/Table/CustomTable.js:
```diff
@@ -73,6 +79,9 @@ const CustomTable = ({ data, columns, totalCount, fetchData, isLoading, navLabel
       bordered
       title={tableHeader}
       loading={isLoading}
+      onRow={(record) => ({
```
this function can take place outside of jsx

melcidar committed:
Data Review Modal - Team Members

--------------------------------------------------------------------
 MAIN
 * added functionality for getting employees to display first and last name
--------------------------------------------------------------------
 OTHER
 * have to edit it so it can take more than one developers
--------------------------------------------------------------------

becarselman commented on src/components/_dataReviewModal/DataReviewModal.js:
Myb we can reuse Elvirs modal. We can pass Component through props and if we have component show overview with that component otherwise show modal for add new item

melcidar committed:
Data Review Modal

--------------------------------------------------------------------
 MAIN
 * added developers' names on modal
--------------------------------------------------------------------
 OTHER
 * project status to be changed
--------------------------------------------------------------------

melcidar committed:
Data Review Modal

--------------------------------------------------------------------
 MAIN
 * edited status and duration info
--------------------------------------------------------------------
 OTHER
 * requested changes, deleting and editing project to be implemented
--------------------------------------------------------------------
